luxjs/nav/sidebar.js

```diff
@@ -166,6 +168,12 @@
                                 lux.querySelector(element, '.content-wrapper').append(inner);
                             else
                                 element.after(inner);
+
+                            if (document.readyState === 'complete') {
+                                $timeout(function() {
+                                    element.removeClass('preload');
+                                });
+                            }
```

Comment:
this does not sound right.
Why are you checking for ``document.readyState``?
What about if it is false? you will never remove that class.